components/mintMe.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { useState } from 'react'
+import { useRouter } from 'next/router'
+import { ethers } from "ethers";
+import Web3Modal from "web3modal";
+import { providerOptions } from './providerOption.js';
+const MintMe = ({ imageToken, abi, contractAddress }) => {
+    const [showPopup, setShowPopup] = useState(false);
+    const [popupText, setPopupText] = useState("");
+
+    const timeout = (delay) => {
+        return new Promise(res => setTimeout(res, delay));
+    }
+
+    const displayPopup = async (text, time) => {
+        setPopupText(text);
+        setShowPopup(true);
+        await timeout(time);
+        setShowPopup(false);
+    }
+    const router = useRouter()
+    const mint = async (token) => {
```

Comment:
this component is super inefficient basically a copy of generate.js but mints the image token instead of 0
will change later